test/system/src/test/java/io/pravega/test/system/ReadWriteAndAutoScaleWithFailoverTest.java

```diff
@@ -94,6 +94,10 @@ public void setup() {
         //get Controller Uri
         controller = new ControllerImpl(controllerURIDirect);
         testState = new TestState();
+        testState.writersListComplete.add(testState.writersComplete);
+        testState.writersListComplete.add(testState.newWritersComplete);
+        testState.writersListComplete.set(0, testState.writersComplete);
```

Comment:
it is to set the index 0 -> writersComplete. It might be redundant here as there is only one element, but i thought it is good to maintain uniformity with other test for easy reference.